consensus/node.go

```diff
@@ -59,43 +56,22 @@ func NewNode(cfg *Config, registry *prometheus.Registry) (*Node, error) {
 
 func (n *Node) Start(ethRPCClient *ethRpc.Client, db *database.Database) error {
 	n.logger.Debug("Creating tendermint ABCI application...")
-	tendermintABCI, err := NewTendermintABCI(db, ethRPCClient, n.metrics)
-	if err != nil {
-		return err
-	}
-
-	n.logger.Debug("Initializing consensus state...")
-	err = tendermintABCI.ResetBlockState()
+	abci, err := NewTendermintABCI(db, ethRPCClient, n.metrics)
 	if err != nil {
 		return err
 	}
 
-	n.logger.Debug("Creating abci server...")
-	proxyLAddr := fmt.Sprintf("tcp://0.0.0.0:%d", n.cfg.proxyListenPort)
-	n.abci, err = tmtServer.NewServer(proxyLAddr, n.cfg.proxyProtocol, tendermintABCI)
-	if err != nil {
-		return err
-	}
-
-	n.logger.Info("Starting ABCI application server...")
-	n.abci.SetLogger(log.NewLogger().With("module", "node-server"))
```

Comment:
Where is the ABCI proxy writing logs after this change?

Reply:
It's just function calls now so the callers are logging now or the methods themselves.